econml/grf/_base_grf.py

```diff
@@ -793,10 +793,13 @@ def predict_full(self, X, interval=False, alpha=0.05):
         """
         if interval:
             point, pred_var = self._predict_point_and_var(X, full=True, point=True, var=True)
+            assert np.isclose(pred_var[pred_var < 0], 0, atol=1e-8).all(), '`pred_var` should not produce large negative values'

```

Comment:
Unfortunately this line is failing our linting step because it's too long, so you'll need to break it up over two lines instead before we can merge.